nosql/userProfile.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "Number",
+  "firstName": "String",
+  "lastName": "String",
+  "headline": "String",
+  "industry": "String",
+  "personalInfo": {"phone" : "String", "email": "String", "adress" : "String", "dob": "String"},
```

Comment:
Not sure about that. mkn t split the address to : Country and zip

Reply:
and leave email and website

Reply:
What do you mean by type ?